src/model/types.ts

```diff
@@ -63,6 +63,32 @@ export function makeApiProblem(
   };
 }
 
+export function mapAuthorizationErrorToApiError(error: unknown): ApiError {
+  return match<unknown, ApiError>(error)
+    .with({ code: ErrorCode.MissingBearer, message: P.string }, (error) =>
+      makeApiProblem(
+        ErrorCode.MissingBearer,
+        400,
```

Comment:
è corretto che siano tutti dei 400? mi aspetterei altri status ma non sto capendo bene il loro codice

(magari comunque potremmo tracciare a parte la ristrutturazione della logica di error handling)

Reply:
@tpetrucciani at the moment I suggest postponing that, it was tracked in a specific task for all error code alignments 👍🏻